models/stg_klaviyo__person.sql

```diff
@@ -42,6 +42,7 @@ final as (
         timezone,
         title,
         updated as updated_at,
+        last_event_date as last_event_date_at,
```

Comment:
is the `date_at` part redundant? wondering if we should leave it as `last_event_date` or rename it to just `last_event_at`

Reply:
good idea, renamed to last_event_at